src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -181,7 +181,7 @@ class SgeBackend extends Backend with SharedFileSystem {
         FailedExecution(new Exception(message), Option(0))
       case (r, _) =>
         postProcess(backendCall) match {
-          case Success(callOutputs) => SuccessfulExecution(callOutputs, r)
+          case Success(callOutputs) => SuccessfulExecution(callOutputs, Seq.empty, r)
```

Comment:
It needs to be a Seq of ExecutionEventEntry. But it's always static and empty here because the SGE backend doesn't generate events.